backstop/tests/side-navigation.html

```diff
@@ -63,159 +70,183 @@
-          class="iui-expand-container"
+        <!-- Expand arrow -->
+        <button
+          class="iui-button iui-expand"
+          role="button"
+          aria-label="Expand menu"
           onclick="handleHidePanel()"
-          tabindex="0"
+          id="test-expand"
         >
-          <!-- Use chevron-left.svg from generic-icons package. -->
+          <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
           <svg
-            viewBox="0 0 16 16"
             class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
           >
-            <path d="M4.7 0L3.3 1.4 9.9 8l-6.6 6.6L4.7 16l8-8z"></path>
+            <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
           </svg>
-        </div>
+        </button>
 
-        <div
-          id="iui-sidenav-content"
-          class="iui-sidenav-content"
-        >
+        <!-- Rest of the tabs -->
+        <div class="iui-content">
           <div
-            id="iui-sidenav-items-container"
-            class="iui-sidenav-items-container"
+            class="iui-content-container"
+            role="tablist"
+            aria-label="Applications"
           >
-            <div
-              id="side-bar-items"
-              class="iui-sidenav-items"
-            >
-              <a
-                class="iui-sidenav-item-container iui-active"
-                tabindex="0"
+            <div class="iui-top">
+              <button
+                class="iui-button iui-large iui-active"
+                role="tab"
+                aria-selected="true"
+                aria-label="Home"
+                aria-controls="home-tab"
               >
-                <!-- Use airplane.svg from generic-icons package. -->
+                <!-- Use home.svg from @itwin/itwinui-icons package. -->
                 <svg
-                  class="iui-sidenav-item-icon"
+                  class="iui-icon"
                   viewBox="0 0 16 16"
+                  aria-hidden="true"
                 >
                   <path
-                    d="M14.003 15.433L12.467 4.735l3.004-3.008s.801-1.204.4-1.605c-.333-.334-1.601.401-1.601.401L11.199 3.6.585 1.994l-.601.602L8.462 6.34l-4.139 4.145-3.07-.2L.183 11.42l2.47.67c-.4 1.336-.2 1.47-.2 1.47s.2.2 1.469-.2l.667 2.473 1.135-1.07-.2-3.075 4.139-4.145 3.738 8.49z"
-                  ></path>
+                    d="m2 14.6s0 .4.3.4h4.7v-4h2v4h4.5c.5 0 .5-.4.5-.4v-6.1l-6-4.5-6 4.5zm12-9.3v-3.3h-2v1.8l-4-2.8-8 5.7 1 1.3 7-5 7 5 1-1.3z"
+                  />
                 </svg>
-                <span class="iui-sidenav-item-caption">Side navigation Item #1</span>
-                <div
-                  class="iui-sidenav-tooltip-container"
-                  role="tooltip"
+                <span class="iui-label">Home</span>
+              </button>
+              <span class="iui-tooltip">Home</span>
+
+              <button
+                class="iui-button iui-large"
```

Comment:
Shouldn't we use borderless button here?

Reply:
After reading this I thought so too, but upon checking we are using the border on the top and bottom.